### DISCUSSION TITLE
Added: LDM notes for April 13th, 2020

### DISCUSSION THREAD

**Discussion by MgSam** [dotnet/csharplang#3366] [General]
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-04-13.md

Didn't see an issue for this so I'll open this one.

**Comment by SingleAccretion** [comment]
I like the decision to only allow `init` for properties a lot. To me this makes the feature much more straightforward and simple (When should I use `init` fields? Are those fields "better" than `readonly`? How are they different anyway?) but still very powerful.

Btw, for people who are interested, `init` has also recently been discussed by the (runtime & BCL) team (see [here](https://github.com/dotnet/runtime/issues/34978)).

**Comment by MgSam** [comment]
I for one am already starting to write TODO comments in my codebase where I want to change properties to `init` after C# 9.0. :smiley:
